### DISCUSSION TITLE
Small typo in the pull request "delete branch" HTML block

### DISCUSSION THREAD

**Discussion by johnjspiteri** [community/community#40810] [Pull Requests]
### Select Topic Area

Bug

### Body

When given the option to "delete branch" after merging a pull request, there is a very small typo

'You’re all set—the Donate branch can be safely deleted.'

Is missing a space before and after the hyphen, and should be:

'You’re all set — the Donate branch can be safely deleted.'

<img width="915" alt="Screen Shot 2022-12-05 at 10 24 50 AM" src="https://user-images.githubusercontent.com/6007361/205601367-719e3b65-dfce-46bb-b8df-690644d5f97b.png">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Johnjspiteri! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
